boxoffice/models/user.py

```diff
@@ -14,6 +14,10 @@ class User(UserBase2, db.Model):
     def __repr__(self):
         return self.fullname
 
+    @property
+    def orgs(self):
```

Comment:
Make this `organizations_owned`, and a function, not a property. It has a non-trivial call cost.